src/elementRoleMap.js

```diff
@@ -82,6 +63,54 @@ const elementRoleMap: TAriaQueryMap<
   },
 };
 
+function ARIARoleRelationConceptAttributeEquals(
```

Comment:
it's rather uncommon to start a function name with a capital letter
```suggestion
function ariaRoleRelationConceptAttributeEquals(
```